Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -2246,6 +2247,36 @@ void FrameLoader::commitProvisionalLoad()
         frame->document() ? frame->document()->url().stringCenterEllipsizedToLength().utf8().data() : "",
         pdl ? pdl->url().stringCenterEllipsizedToLength().utf8().data() : "<no provisional DocumentLoader>", cachedPage.get());
 
+    if (RefPtr document = m_frame->document()) {
+        bool canTriggerCrossDocumentViewTransition = false;
+        RefPtr<NavigationActivation> activation;
+        if (pdl) {
```

Comment:
Not your fault, but IMHO the abbreviation is not nice.

Reply:
Agreed. It's used in a lot of places in this function though, so I won't change it as part of this PR.